content/token-swap.md

```diff
@@ -41,9 +43,12 @@ Creating swap pools with the SPL token swap program really showcases the account
 As we talk through creating a swap pool, we'll assume we're creating a swap pool for two tokens named Token A and Token B. Creating the swap pool with the `spl-token-swap` library is as simple as sending a transaction with an instruction created with the `TokenSwap.createInitSwapInstruction` function. However, there are a number of accounts you need to create or derive beforehand that will be needed when creating that instruction:
 1. **Token swap state account** - holds information about the swap pool
 2. **Swap pool authority** - the account used to sign transactions on behalf of the swap program
+JC: you explain it later, but you can specify that this is a program-derived address based on the token swap state account
 3. **Token accounts for Token A and Token B** - associated token accounts for each of the tokens and the swap pool
+JC: "associated token account" is also a loaded term, so maybe strike the word since it isn't necessary?
 4. **Pool token mint** - the mint for the swap pool's LP-token
 5. **Pool token account** - the token account for the pool token mint
+JC: This is actually the pool token account that receives the first minting of pool tokens, representing the value inside of the token A and B accounts
```

Comment:
@joncinque To clarify, you mean that when the account is first created, some of the LP tokens are minted to this account, but all subsequent LP-tokens are minted directly to the person depositing liquidity?

Reply:
Sweet I'll update some of the language to reflect that